### DISCUSSION TITLE
Docker su NAS QNAP

### DISCUSSION THREAD

**Discussion by mippo9** [indomus/forum#2551] [GENERALE - Discussioni generiche]
Buongiorno,
c'è qualcuno che usa QNAP ?
cioè dopo l'aggiornamento di Container Station alla recente revisione 3.0.3, l'installazione di Docker sembra cambiata, sembra che con il comando "Create" non si riesca più a recuperare nessuna immagine di Home Assistant, quella da reinstallare mensilmente, da Docker Hub.

C'è qualcuno che ne sa qualcosa ?...

**Comment by danidone** [comment]
Ciao riesumo questa discussione poiché da quando container station è passato alla versione 3.0 sono cambiati tutti i parametri per la creazione dei container e soprattutto con home assistant bisogna andare su doker hub a cercare la versione da scaricare che comunque non è così chiaro come e quale scegliere, voi avete delle soluzioni? Io ho Qnap Ts-431x  arm e per ora sono riuscito a scaricare alcune immagini di diverse versioni di HA ma non a farle funzionare.
Grazie a chi mi può dare una mano

**Reply by snakuzzo** [reply]
Non abbiamo più QNAP su cui testare, ma per quanto possa essere cambiata l'interfaccia, immagino che i concetti restino sempre gli stessi...

Quando vai a creare il container, cerca l'immagine `homeassistabt/home-assistant`, clicchi su `Deploy` e poi selezioni quella con tag `stable`
